src/improvements/tasks/eth/001-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,440 @@
+  Who:               0x496286e5eE7758de84Dd17e6d2d97afC2ACE4cc7
+  Contract:          AnchorStateRegistryProxy - Base Mainnet
+  Chain ID:          8453         
+  Raw Slot:          0x0000000000000000000000000000000000000000000000000000000000000001
+  Raw Old Value:     0x0000000000000000000000000000000000000000000000000000000000000000
+  Raw New Value:     0x00000000000000000000000043edb88c4b80fdd2adff2412a7bebf9df42cb40e
+
+  Summary:           Slot 1 is updated to set DisputeGameFactoryProxy address
+  Detail:            Please refer to <i>'Figure 0.2'</i> at the end of this report for the storage layout of AnchorStateRegistry.
+                     <a href="https://github.com/ethereum-optimism/superchain-registry/blob/1a5d7a208cea9b0ea175df1fe71bdc4da7f4c04c/superchain/configs/mainnet/base.toml#L63">0x43edB88C4B80fDD2AdFF2412A7BebF9dF42cB40e</a> is the DisputeGameFactoryProxy address on Base Mainnet.
+  
+----- DecodedStateDiff[9] -----
+  Who:               0x496286e5eE7758de84Dd17e6d2d97afC2ACE4cc7
+  Contract:          AnchorStateRegistryProxy - Base Mainnet
+  Chain ID:          8453        
+  Raw Slot:          0x0000000000000000000000000000000000000000000000000000000000000002
+  Raw Old Value:     0x0000000000000000000000000000000000000000000000000000000000000000
+  Raw New Value:     0x00000000000000000000000049048044d57e1c92a77f79988d21fa8faf74e97e
+
+  Summary:           Slot 2 is updated to set OptimismPortalProxy address
+  Detail:            Please refer to <i>'Figure 0.2'</i> at the end of this report for the storage layout of AnchorStateRegistry.
+                     <a href="https://github.com/ethereum-optimism/superchain-registry/blob/1a5d7a208cea9b0ea175df1fe71bdc4da7f4c04c/superchain/configs/mainnet/base.toml#L58">0x49048044D57e1C92A77f79988d21Fa8fAF74E97e</a> is the OptimismPortalProxy address on Base Mainnet.
+  
+----- DecodedStateDiff[10] -----
+  Who:               0x496286e5eE7758de84Dd17e6d2d97afC2ACE4cc7
+  Contract:          AnchorStateRegistryProxy - Base Mainnet
+  Chain ID:          8453        
+  Raw Slot:          0x0000000000000000000000000000000000000000000000000000000000000004
+  Raw Old Value:     0x0000000000000000000000000000000000000000000000000000000000000000
+  Raw New Value:     0x2e31526db0e371ebf6da52e1f3662801d3352fe8049a573040018c08d94b49cf
+
+  Summary:           Slot 4 updates the <a href="https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/lib/Types.sol#L44">'root'</a> for the <a href="https://github.com/ethereum-optimism/optimism/blob/op-contracts/v2.0.0-rc.1/packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol#L42">startingAnchorRoot</a>
+  Detail:            Please refer to <i>'Figure 0.2'</i> at the end of this report for the storage layout of AnchorStateRegistry.
+                     The 'Raw New Value' for this entry might be different than what is seen in the Tenderly state diff.
```

Comment:
Can you add the line I added [here](https://github.com/ethereum-optimism/superchain-ops/pull/824/files#diff-1eff5a492090ee81aa05524b6a8380071884cc0c8e7b57ab0590b9bb10bb22ffR423) into this PR.